### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by VarLad**
Fixed the link to Pluto Notebooks!

### CODE REVIEW

**Comment by VarLad** [comment]
Also, I don't know how to do it, so can someone please mark it as a draft?
I'll be pushing a new Plotting format!

File: notebooks/README.md [hunk 1]
```diff
@@ -13,81 +13,80 @@ You also need to install the below packages,
...
 #### Intermediate
 
-[5 - More Quantum Gates](https://htmlpreview.github.io/?https://github.com/QuantumBFS/tutorials/blob/master/Notebooks/html/5-more_gates.html)
+[5 - More Quantum Gates](https://htmlpreview.github.io/?https://github.com/QuantumBFS/tutorials/blob/master/notebooks/html/5-more_gates.html)
 - More Single Qubit gates
 - More Multi Qubit gates
 - Some Blocks
 - References - [Yao Documentation](https://docs.yaoquantum.org/)
-- [pdf](https://github.com/QuantumBFS/tutorials/raw/master/Notebooks/pdf/5-more_gates.pdf), [Pluto Notebook](https://raw.githubusercontent.com/QuantumBFS/tutorials/master/Notebooks/Pluto/5-more_gates.jl)
+- [pdf](https://github.com/QuantumBFS/tutorials/raw/master/notebooks/pdf/5-more_gates.pdf), [Pluto Notebook](https://raw.githubusercontent.com/QuantumBFS/tutorials/master/notebooks/Pluto/5-more_gates.jl)
```
Comment (Roger-luo) [inline notebooks/README.md:56]: ah I didn't realize you were using an absolute link, you can just use `pdf/5-more_gates.pdf` etc, but note PDF is not included in the repo, I think we should remote this link.

Comment (VarLad) [reply notebooks/README.md:56]: The pdf part is there for when the Github Actions part is complete

Comment (Roger-luo) [reply notebooks/README.md:56]: yeah but the link is wrong, it will be in the release CDN. let's remote this link for now.

**Pull request closed without merging**